packages/studio-plugin/src/sourcefiles/ComponentFile.ts

```diff
@@ -15,12 +16,14 @@ import { Result } from "true-myth";
 export default class ComponentFile {
   private studioSourceFileParser: StudioSourceFileParser;
   private fileMetadataParser: FileMetadataParser;
+  private dependencyTree: Tree;
 
-  constructor(filepath: string, project: Project) {
+  constructor(filepath: string, project: Project, dependencyTree: Tree) {
```

Comment:
nit: you can just use the `private dependencyTree: Tree` shorthand and it'll create a `private` variable for the class instantiated to the value passed into the constructor

Reply:
done!